pkg/proxy/ipvs/graceful_termination.go

```diff
@@ -150,13 +150,13 @@ func (m *GracefulTerminationManager) GracefulDeleteRS(vs *utilipvs.VirtualServer
 	if err != nil {
 		return err
 	}
-	klog.V(5).Infof("Adding an element to graceful delete rsList: %+v", ele)
+	klog.V(5).InfoS("Adding real server to graceful delete real server list", "realServer", ele)
 	m.rsList.add(ele)
 	return nil
 }
 
 func (m *GracefulTerminationManager) deleteRsFunc(rsToDelete *listItem) (bool, error) {
-	klog.V(5).Infof("Trying to delete rs: %s", rsToDelete.String())
+	klog.V(5).InfoS("Trying to delete real server", "realServer", rsToDelete)
```

Comment:
here I've removed `.String()` as it is a helper function to convert real server to string.
Structured logging does this by default, and I've made every logging statement consistent with having "key" - "value" pairs.
*  log output using klog.V(5).InfoS("Deleting real server", "realServers", rsToDelete.String()). *
`I0921 23:54:10.037778   58346 graceful_termination.go:173] "Deleting real server" realServer="1.1.1.1:80/tcp/10.0.0.1:80"`

* klog.V(5).InfoS("Deleting real server", "realServer", rsToDelete) *
`I0921 23:56:34.439093   58827 graceful_termination.go:173] "Deleting real server" realServer="1.1.1.1:80/tcp/10.0.0.1:80"`

Reply:
Thanks for testing this and providing examples! Looks better!